tests/StorageApiTestCase.php

```diff
@@ -9,11 +9,17 @@
 
 namespace Keboola\Test;
 
+use ErrorException;
```

Comment:
ErrorException, Keboola\StorageApi\Exception a Retry\Policy\CallableRetryPolicy jsou nepouzite

Reply:
Supr dik to som si nevsimol ze som tam nechal